### PR TITLE
[Workers for Platforms] Using ES syntax and adding more information

### PR DESCRIPTION
**Pull request by Maddy-Cloudflare**
PCX-7011

### CODE REVIEW

File: content/cloudflare-for-platforms/workers-for-platforms/get-started/configuration.md [hunk 1]
```diff
@@ -109,33 +109,31 @@ Update the necessary fields and run the following command:
...
-async function handleRequest(request) {
-  return new Response('Hello world');
-}"
+curl -X PUT "https://api.cloudflare.com/client/v4/accounts/<ACCOUNT_ID>/workers/dispatch/namespaces/<NAMESPACE_NAME>/scripts/customer-worker-1" \
+-H "Authorization: Bearer <BEARER_TOKEN>" \
+-H "Content-Type: multipart/form-data" \
+-F 'main_js=@main.js;type=application/javascript+module' -F 'metadata=@metadata.json;type=application/json'
+```
+
+The `main.js` content will look like this:
```
Comment (simonha9) [inline content/cloudflare-for-platforms/workers-for-platforms/get-started/configuration.md]: I suggest showing the main.js and metadata.json files together, we can either:
1. move both here (semantically makes sense but clutters the tutorial)
2. move main.js over to the multipart upload section and have text to ensure that the users use `main_js` and `metadata` field names with "these example files" as the data. (preferred, we already have a multipart section and makes it more reusable in other tutorials, etc)
3. optionally do both - this is probably fine as well

File: content/cloudflare-for-platforms/workers-for-platforms/platform/multipart.md [hunk 2]
```diff
@@ -9,13 +9,20 @@ To perform multipart uploads, include a similar object in your metadata's `bindi
 
 ```json
 {
+    "main_module": "main.js",
```
Comment (simonha9) [inline content/cloudflare-for-platforms/workers-for-platforms/platform/multipart.md]: then after moving main.js over here - it becomes more clear that the metadata.json "main_module" value should correspond to the "main.js" file name - could also be valuable to put some text here to explain the above point ("main_module" key in metadata.json should correspond to the worker file name", or something along those lines)

File: content/cloudflare-for-platforms/workers-for-platforms/get-started/configuration.md [hunk 3]
```diff
@@ -109,33 +109,31 @@ Update the necessary fields and run the following command:
...
--H 'X-Auth-Key: <AUTH_KEY> \
--H 'Content-Type: application/javascript' \
---data "addEventListener('fetch', event => {
-  event.respondWith(handleRequest(event.request));
-})
-
-async function handleRequest(request) {
-  return new Response('Hello world');
-}"
+curl -X PUT "https://api.cloudflare.com/client/v4/accounts/<ACCOUNT_ID>/workers/dispatch/namespaces/<NAMESPACE_NAME>/scripts/customer-worker-1" \
```
Comment (simonha9) [inline content/cloudflare-for-platforms/workers-for-platforms/get-started/configuration.md:112]: Would suggest to tell the users that "JavaScript module syntax is currently only supported through multipart uploads, see here *insert multipart upload docs link here* for an example of uploading a module worker through multipart" (or something along those lines)

**Review by simonha9 — COMMENTED** [review]
Overall looks good, would recommend to add more text about the implicit nature of module uploads through multipart + moving stuff that goes together, together

File: content/cloudflare-for-platforms/workers-for-platforms/get-started/configuration.md [hunk 4]
```diff
@@ -109,35 +109,39 @@ Update the necessary fields and run the following command:
...
+export default {
+  fetch(request) {
+    return new Response('Hello World');
+  },
+};
+```
+
+{{<Aside type="note">}}
+
+You should perform [Multipart uploads](/workers-for-platforms/platform/multipart/) if you want to use Service Workers syntax.
```
Comment (simonha9) [inline content/cloudflare-for-platforms/workers-for-platforms/get-started/configuration.md]: correction: if you want to use *Module Javascript syntax* (Service Workers syntax is the old way)

**Review by simonha9 — APPROVED** [review]
Looks great! Users will have no difficulty setting up WFP (hopefully 😅 )

File: content/cloudflare-for-platforms/workers-for-platforms/get-started/configuration.md [hunk 5]
```diff
@@ -109,35 +109,29 @@ Update the necessary fields and run the following command:
...
-}"
+curl -X PUT "https://api.cloudflare.com/client/v4/accounts/<ACCOUNT_ID>/workers/dispatch/namespaces/<NAMESPACE_NAME>/scripts/customer-worker-1" \
+-H "Authorization: Bearer <BEARER_TOKEN>" \
+-H "Content-Type: multipart/form-data" \
+-F 'main_js=@main.js;type=application/javascript+module' -F 'metadata=@metadata.json;type=application/json'
 ```
 
+{{<Aside type="note">}}
+
+You should perform [Multipart uploads](/cloudflare-for-platforms/workers-for-platforms/platform/multipart/) if you want to use Module Javascript syntax.
```
Comment (deadlypants1973) [inline content/cloudflare-for-platforms/workers-for-platforms/get-started/configuration.md]:
```suggestion
You should perform [Multipart uploads](/cloudflare-for-platforms/workers-for-platforms/platform/multipart/) if you want to use ES Modules syntax.
```

**Review by deadlypants1973 — APPROVED** [review]
PCX

**Pull request merged**